Readme.md

```diff
@@ -3,6 +3,8 @@
 
  X-Response-Time middleware for koa.
 
+___Notice: `koa-response-time@2` supports `koa@2`; if you want to use this module with `koa@1`, please use `koa-response-time@1`.___
```

Comment:
Which version of koa I should write in peerDependencies? As I know latest version is 2.0.0-alpha3.